src/translations/en-US.json

```diff
@@ -841,5 +841,15 @@
         "value": "Start and end values are not included.",
         "comment": "",
         "limit": 0
+    },
+    "mvf.treatNullValuesAsZeroLabel": {
+        "value": "Treat blank values as zero",
+        "comment": "",
+        "limit": 0
+    },
+    "mvf.treatNullValuesAsZeroTooltip": {
+        "value": "Blank values will be included if zero matches the filter condition.",
```

Comment:
I've asked Petr on Slack whether this sentence was proof-read as it confused me when I read it the first time.

Reply:
He let me know the final string will be different...

Reply:
It confused me and Evgenia as well :)) It will be as follows: "If the filter condition includes 0, blank values will be replaced with 0 and included in calculations."